standards/javascript.md

```diff
@@ -188,9 +187,9 @@ to share the same line of code.
 ##### Acceptable
 
 ```js
-var a = 'a';
-var b = 2;
-var i, j;
+const a = 'a';
+const b = 2;
+let i, j;
```

Comment:
How would everyone feel about ditching this one? One variable declaration per line is clearer I think.

Reply:
Yeah, I think we should get more strict about requiring one per line. Also applies to L147.

Reply:
I thought this was weird too, and considered updating it in the other branch I have going. I'll update it there and we can discuss in the code.